.github/workflows/assets-docs.yaml

```
@@ -0,0 +1,107 @@
+name: assets-docs
+
+on:
+  push:
+    branches:
+      - generate-docs
+    # paths-ignore:
+    #   - .github/**
+  # push:
+  #   branches:
+  #     - release
+  #   paths-ignore:
+  #     - .github/**
+  # schedule:
+  #   - cron: '30 6 * * *'
```

Regenerate documentation when the release branch is updated:
```suggestion
    branches:
      - release
    paths-ignore:
      - .github/**
  schedule:
    - cron: '30 8 * * *'
```